### ISSUE TITLE
Add regex support for option -r/-R in the radsniff

### ISSUE THREAD

**Issue by jpereira** [FreeRADIUS/freeradius-server#1118]
Hi,

Suggestion for a good improvement for the great tool.

e.g: 

```
# radsniff -i eth0 -x -r "User-Name =~ /regex/ || Idle-Timeout >= 10"
```

**Comment by arr2036** [comment]
should already work?

**Comment by jpereira** [comment]
nops!

**Comment by jpereira** [comment]
@arr2036 nops. have an idea, how could be the changes?

**Comment by derde** [comment]
This is a regression - probably related to the unlang changes to treat regular expressions as first class syntax elements quoted with slashes (ie. not strings).  In freeradius 2.2 this used to work, with no leading and trailing "/":

    radsniff -r 'User-Name =~ bert'

(matches robert, bertie@domain, etc).  In 3.0.9, you get nothing, even when searching for '.*'

**Comment by derde** [comment]
Matching with `~=` works when the regex is not quoted with slashes, ie.

    radsniff -x -r 'User-Name =~ ..' -c 10

**Comment by arr2036** [comment]
Assuming the fix for this was the same as #1205 if not, please reopen.
